lib/tarantool_server.py

```diff
@@ -245,7 +245,7 @@ def start(self):
         self.generate_configuration()
         if self.script:
             shutil.copy(self.script, self.script_dst)
-            os.chmod(self.script_dst, 511)
+            os.chmod(self.script_dst, 0o777)
```

Comment:
Nit: You touched this line 2 times arcoss there commits. 0777 → 511 → 0o777. Please, use 0o777 right in the first commit.